PaperlessNgx/app.json

```diff
@@ -0,0 +1,10 @@
+{
+  "appid": "6249c08eaa417b9918c69ed2d32ac88b386bc1b2",
```

Comment:
This is the same ID as Paperless used, @KodeStar will this work at all when the name is changed?
Should we maybe keep Paperless and create PaperlessNgx as a total new app with new id?

Reply:
Yeah, create it as a new app with a new id

Reply:
@keriati @KodeStar done